VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -95,9 +96,9 @@ public void onCreate(Bundle savedInstanceState) {
             getAccountTask = new GetAccountTask(getActivity(), profile.username) {
                 @Override
                 protected void onSuccess(@NonNull Account account) throws Exception {
-                    NavigationFragment.this.profileImage = account.getProfileImage();
+                    profileImage = account.getProfileImage();
                     if (null != imageView) {
-                        loadProfileImage(account.getProfileImage(), imageView);
+                        EventBus.getDefault().post(new ProfilePhotoFetchedEvent(profileImage));
```

Comment:
Seems like a potentially better way to do this would be have one "AccountDataLoaded" event that is fired by both the ``getAccount`` and ``updateAccount`` API calls. Then you can just listen for that. Seems like depending on the interactor to fire it, so that a completely different piece of code can listen to it is the wrong level of coupling.